packages/env/lib/config/parse-config.js

```diff
@@ -33,6 +33,8 @@ const HOME_PATH_PREFIX = `~${ path.sep }`;
  * @return {WPServiceConfig} Parsed environment-level configuration.
  */
 module.exports = function parseConfig( config, options ) {
+	console.log( 'options', options );
+	console.log( 'config', config );
```

Comment:
these would go away prior to merge right?

Reply:
Yes.